### PR TITLE
Creating booking without path.

### PR DESCRIPTION
**Pull request by elenicehalma**
The method created, view created, and the simple_form_for. All paths are wrongs.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+
+    if booking.save
+      redirect_to @booking, notice: "Booking was successfully created."
```
Comment (vivien-z) [inline app/controllers/bookings_controller.rb]: maybe should be redirect to bookings_path ?

File: app/views/bookings/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for() do |f|%>
```
Comment (vivien-z) [inline app/views/bookings/new.html.erb]: simple_form_for()  --> need to pass the parameter. in this case maybe should be @booking

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (elenicehalma) [reply app/controllers/bookings_controller.rb]: Now the booking is redirected to theme show.

File: app/views/bookings/new.html.erb (hunk 2 above)
Comment (elenicehalma) [reply app/views/bookings/new.html.erb]: The form is done.

**Pull request merged**